neoload/commands/test_settings.py

```diff
@@ -9,14 +9,16 @@
 from neoload_cli_lib import user_data, cli_exception
 from neoload_cli_lib.name_resolver import Resolver
 
+import logging
+
 __endpoint = "v2/tests"
 __resolver = Resolver(__endpoint)
 
 meta_key = 'settings id'
 
 
 @click.command()
-@click.argument('command', type=click.Choice(['ls', 'create', 'put', 'patch', 'delete', 'use'], case_sensitive=False),
+@click.argument('command', type=click.Choice(['ls', 'create', 'put', 'patch', 'delete', 'use','createoruse'], case_sensitive=False),
```

Comment:
The documentation is missing below on the new createorpatch command

Reply:
Added doc line for 'createorpatch' option in [a9326b3]